src/jobs/models/database/sequelize/sequelizeConnector.js

```diff
@@ -34,18 +34,18 @@ async function insertJob(jobId, jobInfo) {
         proxy_url: jobInfo.proxy_url,
         enabled: jobInfo.enabled,
         debug: jobInfo.debug,
-        webhooks: jobInfo.webhooks ? jobInfo.webhooks.map(webhookUrl => {
-            return { id: uuid(), url: webhookUrl };
-        }) : undefined,
+        // webhooks: jobInfo.webhooks ? jobInfo.webhooks.map(webhookUrl => {
```

Comment:
uncomment

Reply:
done but added a comment that there is still work to do.